crypto/src/main/scala/org/bitcoins/crypto/DLEQUtil.scala

```diff
@@ -15,26 +14,32 @@ object DLEQUtil {
   }
 
   def dleqNonceFunc(
```

Comment:
nit: Drop `Func` from the name since it doesn't return a function

Reply:
I believe this is the appropriate reference in a scaladoc? 

https://github.com/discreetlogcontracts/dlcspecs/blob/d01595b70269d4204b05510d19bba6a4f4fcff23/ECDSA-adaptor.md#secure-nonce-generation